etcd3/locks.py

```diff
@@ -33,14 +33,13 @@ class Lock(object):
 
     lock_prefix = '/locks/'
 
-    def __init__(self, name, ttl=60,
-                 etcd_client=None):
+    def __init__(self, name, ttl=60, etcd_client=None, custom_prefix=''):
```

Comment:
`custom_prefix` can direct rename to `lock_prefix` and use `/locks/` as default value